wave_matching/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.3)
 PROJECT(wave_matching)
 
-SET(CMAKE_BUILD_TYPE RELEASE)
+SET(CMAKE_CXX_FLAGS "-std=gnu++11 ${CMAKE_CXX_FLAGS}")
+SET(CMAKE_BUILD_TYPE release)
```

Comment:
Do not overwrite BUILD_TYPE if already set. Example:
https://github.com/wavelab/okvis_ros/commit/89ab4be8a6c1d956fa7bccd659f829d547891fc9